examples/index.md

```diff
@@ -24,6 +24,7 @@ comments: false
 * [Grape with Paperclip](https://github.com/omarfouad/rails_api_with_ajax_upload): Example of a Grape API mounted on Rails, to upload files with Paperclip & pure AJAX. Details in [this blog post](http://omarfouad.com/blog/2013/08/18/file-upload-api-with-grape-and-rails).
 * [Grape ToDo API](https://github.com/mateuszkosmider/todo-grape-api): A ToDo application.
 * [Grape w/ Rack-OAuth2](https://github.com/balauru/grape-oauth2-sample): Rack app running a Grape API and secured via OAuth2 via rack-oauth2.
+* [Grape Doorkeeper](https://github.com/sethherr/grape-doorkeeper) Rails, Doorkeeper (OAuth2 provider), Grape & Swagger
```

Comment:
Can we make the format of this match the other lines? Include a `:` and a period at the end, please.

Reply:
Yes, sorry about that